include/deal.II/sundials/arkode.h

```diff
@@ -490,8 +493,11 @@ namespace SUNDIALS
      * @param data ARKode configuration data
      * @param mpi_comm MPI communicator
      */
-    ARKode(const AdditionalData &data=AdditionalData(),
-           const MPI_Comm mpi_comm = MPI_COMM_WORLD);
+    ARKode(const AdditionalData &data=AdditionalData()
+#ifdef DEAL_II_WITH_MPI
+                                      , const MPI_Comm mpi_comm = MPI_COMM_WORLD
```

Comment:
Remove this `#ifdef`. The interface should be the same, and both MPI_Comm  and MPI_COMM_WORLD are defined in `deal.II/base/mpi.h`, which is included, also for non MPI compilation.